packages/extensions/openapi-to-cadl/test/arm-compute/cadl-output/models.tsp

```diff
@@ -1,7 +1,6 @@
 import "@typespec/rest";
```

Comment:
why are these in the `cadl-output` folder instead of `tsp-output`?

Reply:
It is just the src-path I'm passing in my test script, users wouldn't get this. I'm updating it to tsp output